src/JitInterface/src/PrimitiveTypeDescriptor.cs

```diff
@@ -0,0 +1,253 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.TypeSystem;
+using System.Diagnostics;
+
+namespace Internal.JitInterface
+{
+    public class PrimitiveTypeDescriptor
+    {
+        internal enum TYPE_ENUM
```

Comment:
This enum is specific to Microsoft PDB files - I expect that we will have a bunch more of similar Microsoft PDB specific enums and structures. Do you agree?

If it is the case, we should invent a namespace for it, something like `namespace ILCompiler.Pdb` and also move this file into a dedicated directory for just the Microsoft PDB specific stuff.